### ISSUE TITLE
Fedora 34: Error when starting container "start.c:core_scheduling:1572 - No such device - Failed to create new core scheduling domain"

### ISSUE THREAD

**Issue by ganto** [ganto/copr-lxc4#18]
After an update from Fedora 33 to Fedora 34 the containers will fail to start with:
```
lxc test 20211205180956.221 ERROR    start - start.c:core_scheduling:1572 - No such device - Failed to create new core scheduling domain
lxc test 20211205180956.222 ERROR    lxccontainer - lxccontainer.c:wait_on_daemonized_start:867 - Received container state "ABORTING" instead of "RUNNING"
lxc test 20211205180956.227 ERROR    start - start.c:__lxc_start:2068 - Failed to spawn container "ipa"
lxc test 20211205180956.227 WARN     start - start.c:lxc_abort:1038 - No such process - Failed to send SIGKILL via pidfd 41 for process 914115
lxc 20211205181001.306 ERROR    af_unix - af_unix.c:lxc_abstract_unix_recv_fds_iov:218 - Connection reset by peer - Failed to receive response
lxc 20211205181001.306 ERROR    commands - commands.c:lxc_cmd_rsp_recv_fds:127 - Failed to receive file descriptors
```
Both Fedora releases used lxd-4.19

**Comment by ganto** [comment]
It seems to be the same issue as described in https://bugs.archlinux.org/task/72496

However, in my case lxc-4.0.11 and lxd-4.19 on Fedora 33 did work fine.

Anyway, I try to get out packages for lxd-4.20 as soon as possible.

**Comment by ganto** [comment]
Issue is fixed with `lxd-4.20-0.1.fc34`
